blivet/__init__.py

```diff
@@ -2170,6 +2179,10 @@ def updateKSData(self):
             cls = getattr(self.ksdata, class_attr)
             data = cls()    # all defaults
 
+            # if this is the raw device of a complementary device use the
+            # complementary device instead.
+            device = next((d for d in complementary_devices if d.raw_device is device), device)
```

Comment:
I'd prefer here a list comprehension assigned to a properly named variable with an ``if`` instead of the combinatio of ``next()`` and a generator. If would be much better readable to me and better matching the comment above it.